docs/data/tree-view/rich-tree-view/selection/selection.md

```diff
@@ -54,3 +74,17 @@ Learn more about the _Controlled and uncontrolled_ pattern in the [React documen
 Use the `onItemSelectionToggle` prop if you want to react to an item selection change:
 
 {{"demo": "TrackItemSelectionToggle.js"}}
+
+## Parent / children selection relationship
```

Comment:
Do you think it's worth adding a demo here on how to do it in user-land?

Reply:
I think it's worth it, yes 👌

Reply:
Done :+1: